spec/support/helpers/generate_renew_token_helper.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "jwt"
+
+module GenerateRenewTokenHelper
```

Comment:
Our other spec helpers are namespaced (eg `Helpers::GenerateRenewTokenHelper`) so I'd suggest doing that here as well 👍